### ISSUE TITLE
<rootDir> in moduleNameMapper

### ISSUE THREAD

**Issue by cocoastorm** [JoinColony/eslint-import-resolver-jest#13]
Hey,

I'm not sure why it can't resolve my files with this config:

`/test/unit/jest.conf.js`
```
const path = require('path');

module.exports = {
  rootDir: path.resolve(__dirname, '../../'),
  moduleNameMapper: {
    '^@/(.*)$': '<rootDir>/src/$1',
  },
```

Is this supported? I'm a little confused with how `eslint-import-resolver-jest` resolves paths if `<rootDir>` is part of the `moduleNameMapper` source.

I'm wildly guessing the resolved path includes the `<rootDir>` twice? Since, if I remove <rootDir> from the moduleNameMapper declaration it works fine.

Also Jest resolves just fine with both ways of configuring the `moduleNameMapper`

**Comment by chmanie** [comment]
Hey, thank you for opening this. Currently this plugin just works with `.json` config files or the `package.json` options for `jest`. I'll try to add support for a more dynamic configuration using `.js` files and node exports.

**Comment by chmanie** [comment]
Turns out we already supported js config files 💃. I'm pretty sure the problem was the rootDir being absolute and addressed it in this PR: #14.
I'll publish a new version quite soon, hope it fixes it for you.
